packages/exchange-analytics/src/models/funding-rate.ts

```diff
@@ -0,0 +1,65 @@
+import { AdapterError } from '@chainlink/ea-bootstrap'
+import { datetime } from '@linkpool/shared'
+
+import { BinanceFundingRatesGetPayload } from '../api/binance/types'
+import { BybitFundingRatesGetPayload } from '../api/bybit/types'
+import { Provider } from '../api/constants'
+import { FtxFundingRatesGetPayload } from '../api/ftx/types'
+
+export interface IFundingRate {
+  nextFunding: {
+    rate: number
+    timestamp: number
+  }
+}
+
+const transformer = {
+  [Provider.BINANCE]: (payload: BinanceFundingRatesGetPayload): IFundingRate => {
+    return {
+      nextFunding: {
+        rate:
+          (Number(payload.markPrice) - Number(payload.indexPrice)) /
+          Number(payload.indexPrice) /
+          24,
+        timestamp: payload.nextFundingTime / 1000,
```

Comment:
These APIs really go to the millisecond. Very well spotted on that timestamps have to be converted to seconds!
Just that for instance you can get `"time": 1663934435009` -> 1663934435.009

We need to floor the division result: `Math.flood(result)`